ci/region.sh

```diff
@@ -12,6 +13,7 @@ case "$1" in
   2)
     echo AWS_DEFAULT_REGION=us-west-1
     echo AWS_REGION=us-west-1
+    echo RACK_BUILD_INSTANCE=m4.xlarge
```

Comment:
I'm guessing it's intentional to not use a build instance in some regions, and to use different sizes here? Maybe a comment stating this intent?